lxd/apparmor/instance_forkproxy.go

```diff
@@ -52,6 +52,9 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   ptrace (read),
   ptrace (trace),
 
+  /etc/machine-id r,
+  {{ .rootPath }}/run/{resolvconf,NetworkManager,systemd/resolve,connman,netconfig}/resolv.conf r,
```

Comment:
On Ubuntu (at least), systemd-resolved managed the following valid targets for `/etc/resolv.conf`:

* `/run/systemd/resolve/stub-resolv.conf`
* `/run/systemd/resolve/resolv.conf`
* `/usr/lib/systemd/resolv.conf`

Reply:
Done